src/main/java/com/checkmarx/flow/custom/ServiceNowTracker.java

```diff
@@ -334,10 +340,10 @@ public String getIssueKey(Issue issue, ScanRequest request) {
     @Override
     public String getXIssueKey(ScanResults.XIssue issue, ScanRequest request) {
         if(flowProperties.isTrackApplicationOnly() || ScanUtils.empty(request.getBranch())){
-            return String.format("%s @ %s", issue.getVulnerability(), issue.getFilename());
+            return String.format(ScanUtils.ISSUE_TITLE_KEY, request.getProduct().getProduct(), issue.getVulnerability(), issue.getFilename());
         }
         else {
-            return String.format("%s @ %s [%s]", issue.getVulnerability(), issue.getFilename(), request.getBranch());
```

Comment:
Same as above, the whole method should be changed to include SCA.
Also, this is another reason why the REST tag search's call shouldn't have the search for the `CX` in the `short_description` parameter, there isn't a `SCA` or `CXSCA` product tag for those issues.